src/mixins/createFormLabelMixin.ts

```diff
@@ -58,6 +58,11 @@ export interface FormLabelMixinProperties {
 	describedBy?: string;
 
 	/**
+	 * ID of a form element associated with the form field
+	 */
+	form?: string;
```

Comment:
this naming confused me, even more so once I read the jsdoc - should probably include `id` in the name.